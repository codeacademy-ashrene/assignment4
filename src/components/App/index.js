import React, { Component } from 'react';
import {
  View, ScrollView, Text, TouchableOpacity,
} from 'react-native';
import Header from '../Header';
import styles from './App.css';
import FormCards from '../FormCards';

export default class App extends Component {
  render() {
    return (
      <View style={styles.container}>
        <Header />
        <ScrollView style={styles.scrollViewContent}>
          <FormCards />
          <FormCards />
          <FormCards />
          <FormCards />
          <FormCards />
        </ScrollView>
        <TouchableOpacity style={styles.addButton} onPress={this.onPressAddButton}>
          <Text style={styles.plus}>+</Text>
        </TouchableOpacity>
      </View>

    );
  }
}

// const styles = StyleSheet.create({
//   container: {
//     flex: 1,
//     justifyContent: 'center',
//     alignItems: 'center',
//     backgroundColor: '#F5FCFF',
//   },
//   welcome: {
//     fontSize: 20,
//     textAlign: 'center',
//     margin: 10,
//   },
//   instructions: {
//     textAlign: 'center',
//     color: '#333333',
//     marginBottom: 5,
//   },
// });
