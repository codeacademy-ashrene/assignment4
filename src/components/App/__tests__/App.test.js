import React from 'react';
import renderer from 'react-test-renderer';
import App from '..';


describe('App', () => {
  it('should render App', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
