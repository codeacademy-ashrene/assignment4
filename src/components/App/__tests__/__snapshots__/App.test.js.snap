// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render App 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#D8E4F0",
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "flex-start",
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
        "flexDirection": "row",
        "justifyContent": "flex-start",
      }
    }
  >
    <Text
      style={
        Object {
          "backgroundColor": "#3F51B5",
          "color": "#FFFFFF",
          "fontSize": 30,
          "height": 80,
          "padding": "5%",
          "width": "100%",
        }
      }
    >
      Awesome Forms
    </Text>
  </View>
  <RCTScrollView
    style={
      Object {
        "marginTop": 100,
      }
    }
  >
    <View>
      <View
        style={
          Object {
            "flex": 1,
            "flexDirection": "column",
            "justifyContent": "center",
            "margin": 20,
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "#FFFFFF",
              "flexGrow": 1,
              "height": 100,
              "justifyContent": "flex-start",
              "padding": "5%",
              "width": 300,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#000000",
                "fontSize": 22,
              }
            }
          >
            Code Academy Feedback 2019
          </Text>
        </View>
        <View
          style={
            Object {
              "borderBottomColor": "#000000",
              "borderBottomWidth": 1,
              "flexGrow": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "#FFFFFF",
              "flexGrow": 1,
              "height": 50,
              "width": 300,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#000000",
                "fontSize": 18,
                "padding": "3%",
              }
            }
          >
            Created On: 26/02/2019
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "flex": 1,
            "flexDirection": "column",
            "justifyContent": "center",
            "margin": 20,
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "#FFFFFF",
              "flexGrow": 1,
              "height": 100,
              "justifyContent": "flex-start",
              "padding": "5%",
              "width": 300,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#000000",
                "fontSize": 22,
              }
            }
          >
            Code Academy Feedback 2019
          </Text>
        </View>
        <View
          style={
            Object {
              "borderBottomColor": "#000000",
              "borderBottomWidth": 1,
              "flexGrow": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "#FFFFFF",
              "flexGrow": 1,
              "height": 50,
              "width": 300,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#000000",
                "fontSize": 18,
                "padding": "3%",
              }
            }
          >
            Created On: 26/02/2019
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "flex": 1,
            "flexDirection": "column",
            "justifyContent": "center",
            "margin": 20,
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "#FFFFFF",
              "flexGrow": 1,
              "height": 100,
              "justifyContent": "flex-start",
              "padding": "5%",
              "width": 300,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#000000",
                "fontSize": 22,
              }
            }
          >
            Code Academy Feedback 2019
          </Text>
        </View>
        <View
          style={
            Object {
              "borderBottomColor": "#000000",
              "borderBottomWidth": 1,
              "flexGrow": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "#FFFFFF",
              "flexGrow": 1,
              "height": 50,
              "width": 300,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#000000",
                "fontSize": 18,
                "padding": "3%",
              }
            }
          >
            Created On: 26/02/2019
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "flex": 1,
            "flexDirection": "column",
            "justifyContent": "center",
            "margin": 20,
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "#FFFFFF",
              "flexGrow": 1,
              "height": 100,
              "justifyContent": "flex-start",
              "padding": "5%",
              "width": 300,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#000000",
                "fontSize": 22,
              }
            }
          >
            Code Academy Feedback 2019
          </Text>
        </View>
        <View
          style={
            Object {
              "borderBottomColor": "#000000",
              "borderBottomWidth": 1,
              "flexGrow": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "#FFFFFF",
              "flexGrow": 1,
              "height": 50,
              "width": 300,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#000000",
                "fontSize": 18,
                "padding": "3%",
              }
            }
          >
            Created On: 26/02/2019
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "flex": 1,
            "flexDirection": "column",
            "justifyContent": "center",
            "margin": 20,
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "#FFFFFF",
              "flexGrow": 1,
              "height": 100,
              "justifyContent": "flex-start",
              "padding": "5%",
              "width": 300,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#000000",
                "fontSize": 22,
              }
            }
          >
            Code Academy Feedback 2019
          </Text>
        </View>
        <View
          style={
            Object {
              "borderBottomColor": "#000000",
              "borderBottomWidth": 1,
              "flexGrow": 1,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "#FFFFFF",
              "flexGrow": 1,
              "height": 50,
              "width": 300,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#000000",
                "fontSize": 18,
                "padding": "3%",
              }
            }
          >
            Created On: 26/02/2019
          </Text>
        </View>
      </View>
    </View>
  </RCTScrollView>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "alignSelf": "flex-end",
        "backgroundColor": "#3F51B5",
        "borderColor": "#ffffff",
        "borderRadius": 100,
        "bottom": 40,
        "height": 80,
        "opacity": 1,
        "position": "absolute",
        "right": 20,
        "width": 80,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#FFFFFF",
          "fontSize": 55,
        }
      }
    >
      +
    </Text>
  </View>
</View>
`;
